src/gen_dragonfruit/gd_dev_apogee_sp215.cpp

```diff
@@ -21,9 +26,8 @@ uint32_t gd_dev_apogee_sp215_read(void){
         value += Wire.read();
     }
     Wire.endTransmission();
-    /* Analog to digital conversion with 16 bit-resolution */   
-    /* Divide by 0xFFFF to convert 16-bit ADC reading, 5 V is reference voltage */
-    /* Physical voltage divider implemented to prevent saturation of signal. Apogee SP215 has a maximum output voltage of 5V but the MCP3425 only allows a max input of 2.048V. Voltage divider over .973kohm and 2.167kohm resistor used to decrease amplitude of signal from Apogee. Use a scaling factor of 3.22713257965 to account for this voltage divider and calculate the value of the original signal. */
+    
+    /* Analog to digital conversion with 16 bit-resolution. Multiply by 5V reference voltage then divide by 0xFFFF to convert the 16-bit ADC reading to voltage. The Apogee SP215 sensor has a maximum output voltage of 5V but the MCP3425 ADC converter only allows a max input reading of 2.048V. To prevent a saturated signal reading, A physical voltage divider over R1 (.973 kohm) and R2 (2.167 kohm) is used to decrease the signal amplitude from the Apogee SP215. To account for this voltage divider, a scaling factor of 3.227 is used. */
```

Comment:
Can you limit each line to less than 80 characters (break it up into multiple lines)?